content/overview/workflow/import-review.md

```diff
@@ -0,0 +1,29 @@
+---
+title: import & review
+id: import-review
+draft: false
+weight: 20
+author: "people"
+---
+
+Before you can do anything in darktable you must first add some images files to the library using the [import](../../module-reference/utility-modules/lighttable/import.md) module in the [lighttable view](../../lighttable/_index.md). This will create entries for your images in darktable's library database so that it can keep track of the changes you make. There are three ways to import images, each accessible through buttons in the import module:
+
+add to library
+: This option adds images to the library without copying or moving them -- your original files will stay in their current location and will not be altered. On import, darktable will read the metadata from the image files and any accompanying [XMP sidecar file](../../overview/sidecar-files/_index.md). If an image has already been added to the database, any updates you have made to the sidecar file will be loaded.
+
+copy & import
+: This is the same as "add to library" but instead of loading images from their current location, darktable first copies them to a new location (following the file naming pattern defined in [preferences > import](../../preferences-settings/import.md)) and then adds the _copied_ images to the library -- the original images are not changed or moved.
+
+copy & import from camera
+: To import images from a camera, first connect the camera to your system with a USB cable (if your camera is auto-mounted by your system, you will need to un-mount it before it can be accessed by darktable). If you don't see your camera listed in the import module, press the "scan for devices" button. Once your camera is detected the import module should offer the ability to _copy & import_ images from the camera. Clicking the "copy & import" button physically copies the selected images from the camera into a specified directory (following the file naming pattern defined in [preferences > import](../../preferences-settings/import.md)) and then adds the copied images to the library.
```

Comment:
```suggestion
: Connect the camera to your system with a USB cable (if your camera is auto-mounted by your system, you will need to un-mount it before it can be accessed by darktable). If you don't see your camera listed in the import module, press the "scan for devices" button. Once your camera is detected the import module should offer the ability to _copy & import_ images from the camera. Clicking the "copy & import" button physically copies the selected images from the camera into a specified directory (following the file naming pattern defined in [preferences > import](../../preferences-settings/import.md)) and then adds the copied images to the library.
```